ledfx/nowplaying/providers/base.py

```
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+from typing import Callable, Protocol
+from collections.abc import Awaitable
+
+from ledfx.nowplaying.models import NowPlayingTrack
+
+
+class NowPlayingProvider(Protocol):
+    """Minimal contract for now-playing metadata providers."""
+
+    async def start(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)